### PR TITLE
Fix some errors in List module documentation

### PR DESCRIPTION
**Pull request by tokayy**
Mostly just minor grammatical errors except for the `List.chunkBySize` summary which is hard to make sense of.

### CODE REVIEW

**Comment by tokayy** [comment]
@dotnet-policy-service agree

17 Nov 2023 at 00:51 by ***@***.***:

File: src/FSharp.Core/list.fsi [hunk 1]
```diff
@@ -243,7 +243,7 @@ module List =
     [<CompiledName("ChunkBySize")>]
     val chunkBySize: chunkSize:int -> list:'T list -> 'T list list
 
-    /// <summary>For each element of the list, applies the given function. Concatenates all the results and return the combined list.</summary>
+    /// <summary>For each element of the list, apply the given function. Concatenates all the results and return the combined list.</summary>
```
Comment (abonie) [inline src/FSharp.Core/list.fsi:246]: Based on some other summaries I see in this PR, seems like they are all written in third person.
```suggestion
    /// <summary>For each element of the list, applies the given function. Concatenates all the results and returns the combined list.</summary>
```
Not sure about the comma. Maybe just "Applies the given function to each element of the list" sounds more natural, but I don't know English well enough to nitpick here.

**Review by abonie — APPROVED** [review]

**Review by psfinaki — APPROVED** [review]
Thanks for your first contribution @tokayy! Keep on :)

File: src/FSharp.Core/list.fsi (hunk 1 above)
Comment (tokayy) [reply src/FSharp.Core/list.fsi]: Looking back at it again, I think I agree with you. I'll leave this one out for now, although it still reads a bit odd to me.

**Pull request merged**